templates/CRM/PivotReport/Page/PeopleReport.tpl

```diff
@@ -16,7 +16,16 @@
     </fieldset>
   </form>
 </div>
-
+<div id="pivot-report-record-filter-form" class="hidden">
+  <form>
+    <label>Headcount on date:</label><br />
+    <input name="headCountOnDate" class="crm-ui-datepicker" />
```

Comment:
What if we need to add another filter here in future, and that type is not date?
we can do this like below
HTML
```html
<input name="headCountOnDate" class="crm-ui-datepicker" filter-type="date"/>
<input name="displayName" class="" filter-type="text"/>
```

JS
```js
initRecordFilterDefaultValues: function() {
  if (filterType === 'date') { current implementation}
  if (filterType === 'text') { we can add later }
}
```

Reply:
As discussed the name of the function was changed to `initRecordFilterDefaultDateValues` since this function only adds default values to date filters.